gatein/src/main/java/org/exoplatform/platform/qa/ui/gatein/pageobject/UserAndGroupManagement.java

```diff
@@ -540,11 +540,11 @@ public void editUserInfo_UserProfileTab(String givenName,
     if (language != null && language != "") {
       evt.select(ELEMENT_LANGUAGE, language);
     }
-    evt.click(ELEMENT_SAVE_BUTTON);
-    evt.waitForElementNotPresent(ELEMENT_SAVE_BUTTON);
-    evt.waitForMessage(ELEMENT_MSG_UPDATE_USER_PROFILE);
-    evt.click(GateinLocator.ELEMENT_CLOSE_MESSAGE);
-    evt.waitForElementNotPresent(GateinLocator.ELEMENT_CLOSE_MESSAGE);
+    ELEMENT_SAVE_BUTTON.click();
+    ELEMENT_SAVE_BUTTON.waitUntil(Condition.not(Condition.visible),Configuration.timeout);
+    ELEMENT_MSG_UPDATE_USER_PROFILE.waitUntil(Condition.visible,Configuration.timeout);
+    ELEMENT_CLOSE_MESSAGE.click();
+    ELEMENT_CLOSE_MESSAGE.waitUntil(Condition.not(Condition.visible),Configuration.timeout);
```

Comment:
Can not run test because of wrong declaration of these elements (ELEMENT_SAVE_BUTTON, ELEMENT_MSG_UPDATE_USER_PROFILE, ELEMENT_CLOSE_MESSAGE): their type must be Selenide not String